### DISCUSSION TITLE
how to use expand in extend with js

### DISCUSSION THREAD

**Discussion by nikhil-varma-mudunuri** [pocketbase/pocketbase#3967] [Q&A] [answered]
`  record = $app.dao().findRecordsByFilter(
      "mc", collection
      "match_id = '${}'"
    );`
    
    how to use expand option for all the filtered records

**Comment by ganigeorgiev** [comment, marked as answer]
I'm not sure that I understand your code sample and what is the desired outcome, but for expanding relations programmatically you can check https://pocketbase.io/docs/js-records/#programmatically-expanding-relations.

If this doesn't help, please try to elaborate a little more on your use case and what you are trying to do.
